opentelemetry-prometheus/src/lib.rs

```diff
@@ -175,6 +204,22 @@ impl ExporterBuilder {
     }
 }
 
+/// Config for prometheus exporter
+#[derive(Debug, Default)]
+pub struct ExporterConfig {
+    /// disable the otel_scope_info metric and otel_scope_ labels.
+    disable_scope_info: bool,
+}
+
+impl ExporterConfig {
+    /// Set disable_scope_info for [`ExporterConfig`].
+    /// It's the flag to disable the otel_scope_info metric and otel_scope_ labels.
+    pub fn disable_scope_info(mut self) -> Self {
```

Comment:
I know we should default to `false` here but still may be worth to allow the user to pass a bool to change the behavior

Reply:
So does `otel_scope_info` need to be disabled by default?

Reply:
So based on the spec it should be on by default https://github.com/open-telemetry/opentelemetry-specification/blob/0d9081669c0c087c092bf35fe5c5652c2b505b94/specification/compatibility/prometheus_and_openmetrics.md#instrumentation-scope-1
but I agree that  `with_scope_info`  would be  more inline with how the interfaces are working.

Reply:
> So based on the spec it should be on by default https://github.com/open-telemetry/opentelemetry-specification/blob/0d9081669c0c087c092bf35fe5c5652c2b505b94/specification/compatibility/prometheus_and_openmetrics.md#instrumentation-scope-1 but I agree that `with_scope_info` would be more inline with how the interfaces are working.

I changed `disable_scope_info(mut self) -> Self` to `with_scope_info(mut self, enabled: bool) -> Self`, and let the default value be true.